lib/network.js

```diff
@@ -0,0 +1,143 @@
+var util = require('./util');
+
+/**
+ * Represents an network
+ * @param {Object} modem docker-modem
+ * @param {String} id  Network's id
+ */
+var Network = function(modem, id) {
+  this.modem = modem;
+  this.id = id;
+};
+
+/**
+ * Get
+ * @param  {Function} callback Callback with data stream.
+ */
+
+Network.prototype.get = function(callback) {
+    if (typeof callback === 'function') {
+      var opts = {
+        path: '/networks/' + this.id,
+        method: 'GET',
+        statusCodes: {
+          200: true,
+          404: 'no such network',
+          500: 'server error'
+        }
+      };
+
+      this.modem.dial(opts, function(err, data) {
+        callback(err, data);
+      });
+    } else {
+      return JSON.stringify({Id: this.id});
+    }
+}
+
+/**
+ * Inspect
+ * @param  {Function} callback Callback, if specified Docker will be queried.
+ * @return {Object}            Id only if callback isn't specified.
+ */
+Network.prototype.inspect = function(callback) {
+  if (typeof callback === 'function') {
+    var opts = {
+      path: '/networks/' + this.id,
+      method: 'GET',
+      statusCodes: {
+        200: true,
+        404: 'no such network',
+        500: 'server error'
+      }
+    };
+
+    this.modem.dial(opts, function(err, data) {
+      callback(err, data);
+    });
+  } else {
+    return JSON.stringify({Id: this.id});
```

Comment:
I don't know. I found the same pattern in volume.inspect, container.inspect and image.inspect